app/queries/base_path_state.rb

```diff
@@ -0,0 +1,71 @@
+module Queries
+  module BasePathState
+    extend ArelHelpers
+
+    def self.conflict_with_draft_content(content_item_id, base_path)
```

Comment:
Can we extract the common query parts from these two methods into some helper method or something?